src/components/Landing/styles.scss

```
@@ -0,0 +1,77 @@
+// Mobile portrait
+.wrapper {
+	display: flex;
+	width: 100%;
+	height: 100%;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	background-color: #FF60D7;
+	font-size: 1.15rem;
+}
+
+	.heading, .subheading {
+		font-family: var(--font_family__heading);
+	}
+
+	.heading {
+		margin-bottom: 0.6em;
+		font-size: 3.33em;
+		text-transform: uppercase;
+	}
+
+	.subheading {
+		margin-bottom: 1em;
+		font-size: 2.5em;
+	}
+
+	.body {
+		text-align: center;
+		font-size: 1em;
+		font-family: var(--font_family__body);
+
+		p+p {
```

```suggestion
		.paragraph + .paragraph {
```
Just gotta update `renderUtils`